File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    sizes.map(v=> v.sort((a,b)=> a-b))
    let max1 = Math.max(...sizes.map(v=>v[0]))
    let max2 = Math.max(...sizes.map(v=>v[1]))
    
    return max1*max2    
} 

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
import java.util.*;

class Solution {
    public int solution(int[][] sizes) {
        int max1 = 0;
        int max2 = 0;
        for (int[] size : sizes) {
            Arrays.sort(size);
            if (size[0] > max1) {
                max1 = size[0];
            }
            if (size[1] > max2) {
                max2 = size[1];
            }
        }
        return max1 * max2;
    }
}